eve/workers/pod.yaml

```diff
@@ -12,6 +12,18 @@ spec:
     hostnames:
     - "bucketwebsitetester.s3-website-us-east-1.amazonaws.com"
   containers:
+    {% if vars.env.S3METADATA is defined and vars.env.S3METADATA == "mongodb" -%}
```

Comment:
Does changing the order help out with the startup? Would it be quicker to start up if we hosted the mongo image on the google repository?

Reply:
I thought it was going to. When I was trying to debug why the CI was not working I ran:
```bash
$ kubectl describe pod ci-pod
# I saw there was some kind of order, So I thought maybe this could help
  Normal  Pulled                 2m    kubelet, gke-eve-prod-worker-pool-d5074c06-qlqb  Container image "gcr.io/eve-prod/aggressor_8cb7:f693042075686cb6c0ff2fd843a93eb370bc64c16007374df1ed2fbbac29a5ce" already p
resent on machine                                                                                                                 
  Normal  Created                2m    kubelet, gke-eve-prod-worker-pool-d5074c06-qlqb  Created container                         
  Normal  Started                2m    kubelet, gke-eve-prod-worker-pool-d5074c06-qlqb  Started container                         
  Normal  Pulling                2m    kubelet, gke-eve-prod-worker-pool-d5074c06-qlqb  pulling image "gcr.io/eve-prod/s3_3a52:e8fce2242b03c67b23b372f83b865449d55eb8a08bb5daf35c10eccc48365cef"
  Normal  Pulled                 2m    kubelet, gke-eve-prod-worker-pool-d5074c06-qlqb  Successfully pulled image "gcr.io/eve-prod/s3_3a52:e8fce2242b03c67b23b372f83b865449d55eb8a08bb5daf35c10eccc48365cef"
  Normal  Created                2m    kubelet, gke-eve-prod-worker-pool-d5074c06-qlqb  Created container                         
  Normal  Started                2m    kubelet, gke-eve-prod-worker-pool-d5074c06-qlqb  Started container                        
  Normal  Pulled                 2m    kubelet, gke-eve-prod-worker-pool-d5074c06-qlqb  Container image "scality/ci-mongo:3.4" already present on machine
  Normal  Created                2m    kubelet, gke-eve-prod-worker-pool-d5074c06-qlqb  Created container                        
  Normal  Started                2m    kubelet, gke-eve-prod-worker-pool-d5074c06-qlqb  Started container                         
  Normal  Pulled                 2m    kubelet, gke-eve-prod-worker-pool-d5074c06-qlqb  Container image "redis:alpine" already present on machine
  Normal  Created                2m    kubelet, gke-eve-prod-worker-pool-d5074c06-qlqb  Created container                        
  Normal  Started                2m    kubelet, gke-eve-prod-worker-pool-d5074c06-qlqb  Started container
```
But, no luck at the end.

And regarding the hosting of the mongo image, well, it might help, a little bit. But that won't fix the fundamental as I think this "bug" not only affect the CI deployment but also helm charts and all. We're lucky that kubernetes just restart the pod, which is not the case in the CI.